app/styles/components/_nav.scss

```diff
@@ -77,6 +77,32 @@ $nav-break-mobile: 798px;
   background-image: url('../images/hamburger@2x.png');
 }
 
+.Nav.Nav--onTopIndex {
+  background: none;
+  box-shadow: none;
+  justify-content: flex-end;
```

Comment:
This is intentional. The Nav class has "justify-content: space-between" which would move the "Officers & Board" link to the left side of the screen as the logo is no longer present in the Index page